src/cogs/interactions/slash/utils.py

```diff
@@ -446,6 +446,59 @@ async def fetch_data():
                 with open("faq.json", "r") as file:
                     data = json.load(file)
                 return data
+
+    @staticmethod
+    async def invite_user_to_github_org(github_username: str, org: str = None, team_slug: str = None) -> dict:
+        github_token=os.getenv("GITHUB_TOKEN")
+        github_org_name=os.getenv("GITHUB_ORG_NAME")
+        github_slug=os.getenv("GITHUB_SLUG")
+
+        if not github_token:
+            return {"success": False, "message": "GitHub token not configured", "status_code": 500}
+        headers = {
+          "Accept": "application/vnd.github+json",
+          "Authorization": f"Bearer {github_token}",
+          "X-GitHub-Api-Version": "2022-11-28",
+          "User-Agent": "PESU-Discord-Bot"
+        }
+        try:
+            async with httpx.AsyncClient(headers=headers) as client:
+                org_invite_url = f"https://api.github.com/orgs/{github_org_name}/invitations"
+                org_payload = {
+                    "invitee_id": None,
+                    "email": None,
+                    "role": "direct_member"
+                }
+                user_response = await client.get(f"https://api.github.com/users/{github_username}")
+                if user_response.status_code != 200:
+                    return {"success": False, "message": f"GitHub user '{github_username}' not found", "status_code": 404}
+                user_data = user_response.json()
+                org_payload["invitee_id"] = user_data["id"]
+                org_response = await client.post(org_invite_url, json=org_payload)
+
+                if org_response.status_code == 404: 
+                    return {"success": False, "message": f"Failed to invite to org: {org_response.text}", "status_code": org_response.status_code}
+                
```

Comment:
do you need to explicitly send an invite to them if they're not part of the org yet? GitHub documentation (from this [link](https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#add-or-update-team-membership-for-a-user)) says this:
> An organization owner can add someone who is not part of the team's organization to a team. When an organization owner adds someone to a team who is not an organization member, this endpoint will send an invitation to the person via email. This newly-created membership will be in the "pending" state until the person accepts the invitation, at which point the membership will transition to the "active" state and the user will be added as a member of the team.